contrib/static-web-server/patches/0003-generate-man-page-and-completions.patch

```diff
@@ -0,0 +1,219 @@
+From 6d3747bb733837686df0f0f655bf0cd7ae774346 Mon Sep 17 00:00:00 2001
```

Comment:
send this upstream and only backport after it's actually merged

Reply:
It's not merged *yet*, but I'm happy with just keeping this open until it is merged upstream. Should happen soon.

Reply:
Patch is merged upstream now.